std/container/util.d

```diff
@@ -140,3 +140,37 @@ unittest
     auto rbtmin = make!(RedBlackTree, "a < b", false)(3, 2, 2, 1);
     assert(equal(rbtmin[], [1, 2, 3]));
 }
+
+package:
+/**
+ *String representation of the container
+ */
+mixin template forwardToStringTo(alias symbol)
+{
+    string toString() {
```

Comment:
Although nullary `toString` like this is important to include for classes so it overrides `Object.toString`, please add sink-based `toString`.